tests/core/sprout_checks/test_check_resource_properties_sprout.py

```diff
@@ -0,0 +1,157 @@
+from pathlib import Path
+
+from pytest import fixture, mark, raises
+
+from seedcase_sprout.core.checks.check_error import CheckError
+from seedcase_sprout.core.properties import ResourceProperties
+from seedcase_sprout.core.sprout_checks.check_resource_properties import (
+    check_resource_properties,
+)
+from seedcase_sprout.core.sprout_checks.failed_check_error import FailedCheckError
+from seedcase_sprout.core.sprout_checks.get_blank_value_for_type import (
+    get_blank_value_for_type,
+)
+from seedcase_sprout.core.sprout_checks.required_fields import (
+    RESOURCE_SPROUT_REQUIRED_FIELDS,
+)
+
+
+@fixture
+def properties():
+    return ResourceProperties(
+        name="resource-1",
+        path=str(Path("resources", "1", "data.parquet")),
+        title="Resource 1",
+        description="A resource.",
+    ).compact_dict
+
+
+@mark.parametrize("check_required", [True, False])
+def test_passes_full_resource_properties(properties, check_required):
+    """Should pass if all required fields are present and correct."""
+    assert (
+        check_resource_properties(properties, check_required=check_required)
+        == properties
+    )
+
+
+@mark.parametrize(
+    "field",
+    RESOURCE_SPROUT_REQUIRED_FIELDS.keys(),
+)
+def test_fails_if_required_field_missing(properties, field):
+    """Should fail if a required field is missing."""
```

Comment:
Same with naming of tests here as in the other PRs (sorry 😁 ) 
```suggestion
def test_raises_error_if_required_field_missing(properties, field):
    """Should raise FailedCheckError if a required field is missing."""
```

Reply:
Will do! (They will all be like this haha)

Reply:
Haha, yes, I figured 😅  But still wanted to mention it.